packages/connector-jsdom/src/connector.ts

```diff
@@ -362,21 +367,33 @@ export default class JSDOMConnector implements IConnector {
         return this._fetchUrl(parsedTarget, customHeaders);
     }
 
-    private killProcess = (runner: ChildProcess) => {
+    private killProcess(runner: ChildProcess) {
         try {
             runner.kill('SIGKILL');
+
+            this._subprocesses.delete(runner);
```

Comment:
Not that it really matters, but isn't this redundant given the call to `this._subprocesses.clear()` in `killAllSubprocesses`?

Reply:
I think I'd delete the `this._subprocesses.clear()` in `killAllSubprocesses` instead of this one